ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -94,12 +94,21 @@ module('Acceptance | aws | configuration', function (hooks) {
       this.flashSuccessSpy.calledWith('The backend configuration saved successfully!'),
       'Success flash message is rendered'
     );
+    // server.get here because we want it after they have pressed saved.
+    this.server.get(`${path}/config/root`, () => {
```

Comment:
nit: I prefer to have the server configs at the top of the test so it's clear what we expect in the test from reading the top of it. 

Separately, if it's in the code here because we want the behavior to change mid-test, that feels like brittle behavior to rely on. Usually when I want the behavior of a mocked endpoint to change, I do something like this: 

```
let rootReadCount = 0;
this.server.get(`${path}/config/root`, () => {
  rootReadCount++;
  if (rootReadCount > 1) {
    // on first call, return this
    return { ... }
  }
  // otherwise return this
  return { ... }
}  
```

Reply:
Alright, after trying several different approaches, i can't get this to work. If I move the get that high up it never gets hit. I suspect it has something to do with mirage and not being able to access that getter high up in the test. Maybe because it won't access a this.server interruption above where I've changed routes?